commands/initial/templates/package.json

```diff
@@ -15,7 +15,9 @@
     "prebuild": "rimraf dist out-tsc",
     "start": "webpack-dev-server --open --config ./webpack/webpack.dev.js",
     "tagVersion": "np --no-publish",
-    "test": "node ./tasks/test"
+    "test": "node ./tasks/test",
+    "pree2e": "webdriver-manager update",
+    "e2e": "protractor"
```

Comment:
It's less about my concerns but more about simplicity. You can of course do what you describe but that somehow pulls all the test related parameters into your ./tasks/test script.

But NPM enables one to append configuration by appending it after a double dash after run command e.g. `npm run e2e -- --seleniumServerAddress ADDRESS` which gets resolved to `protractor --seleniumServerAddress ADDRESS` which then overwrites the default seleniumServerAddress (or the one from the configuration) with the new value ADDRESS.
(That's why most of the tools have the precedence in the following order: CLI > configuration > user-dotfile > default value)
 
This powermove of NPM enables you (the developer) to just run `npm run e2e` locally to run the tests (in this case: default) selenium server.  In your CI environment however you then run `npm run e2e -- --seleniumServerAddress ADDRESS` to adjust your build script to the CI environment.